fluent-tests/prepare-tests.bat

```diff
@@ -56,6 +56,9 @@ if %errorlevel% neq 0 exit /b %errorlevel%
 REM conflict property name from 2 x-ms-flatten in LabDetails and LabProperties
 CALL autorest --version=%AUTOREST_CORE_VERSION% %FLUENT_ARGUMENTS% --input-file=https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/education/resource-manager/Microsoft.Education/preview/2021-12-01-preview/education.json --namespace=com.azure.mgmttest.education
 if %errorlevel% neq 0 exit /b %errorlevel%
+REM do not flatten if polymorphic in DevicePropertiesFormat
+CALL autorest --version=%AUTOREST_CORE_VERSION% %FLUENT_ARGUMENTS% --input-file=https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/hybridnetwork/resource-manager/Microsoft.HybridNetwork/stable/2021-05-01/device.json --namespace=com.azure.mgmttest.hybridnetwork
```

Comment:
I might also need to convert it to PS. But again, later, after we got our holidays :-)